### PR TITLE
WET-255 - GC Task Success Survey: Survey Referrer value being cut-off

### PR DESCRIPTION
**Pull request by Ricokola**
- Finding an alternative for providing the referrer path following the domain name.
- Using `window.location.pathname` to fetch everything following the domain name and add it pass it via a URL parameter.
- From the task success survey, fetching the URL parameter and concatenate it  to the domain name in `document.referrer`.

Working example: https://ricokola.github.io/invitation-manager/docs/test-IM/test-eng.html?logim=1&im_scope=Page&im_surveyid=4&im_nocookiecheck=1&im_nodatecheck=1

### CODE REVIEW

**Comment by Ricokola** [comment]
@GormFrank or @duboisp Can you please review this PR.

Note: This is set for the working example. As soon it's approved, I will adjust to reflect the Service Canada example.

File: docs/config.JSON [hunk 1]
```diff
@@ -1,3 +1,3 @@
-{ "dbURL": "/invitation-manager/docs/im.json",
+{ "dbURL": "/invitation-manager/docs/im3.json",
```
Comment (GormFrank) [inline docs/config.JSON]: Please keep im.json here

File: docs/im2.json [hunk 2]
```diff
@@ -0,0 +1,187 @@
+﻿{
```
Comment (GormFrank) [inline docs/im2.json]: No need for this file, just update im.json if needed

File: docs/im3.json [hunk 3]
```diff
@@ -0,0 +1,187 @@
+﻿{
```
Comment (GormFrank) [inline docs/im3.json]: No need for this file.

File: docs/test-IM/test-eng.html [hunk 4]
```diff
@@ -11,8 +11,8 @@
 	<noscript>
 		<link rel="stylesheet" href="https://wet-boew.github.io/themes-dist/GCWeb/wet-boew/css/noscript.min.css" />
 	</noscript>
-	<!-- Invitation Manager CSS Files -->
-	<link rel="stylesheet" href="https://servicecanada.github.io/invitation-manager/src/Overlay.css">
+	<!-- Invitation Manager CSS Files  -->
+	<link rel="stylesheet" href="https://ricokola.github.io/invitation-manager/src/Overlay.css">
```
Comment (GormFrank) [inline docs/test-IM/test-eng.html]: Please keep the servicecanada URL

File: docs/test-IM/test-eng.html [hunk 5]
```diff
@@ -216,12 +216,12 @@ <h2 class="wb-inv">About this site</h2>
 	<!-- used to check for the path of config.JSON file. 
 		If not declared, config.JSON is expected to be in the same directory as InvitationManger.js -->
 	<script>
-		window.imConfigPath = "https://servicecanada.github.io/invitation-manager/docs/";
+		window.imConfigPath = "https://ricokola.github.io/invitation-manager/docs/";
```
Comment (GormFrank) [inline docs/test-IM/test-eng.html]: Please keep the servicecanada URL, and update the docs location if needed

File: docs/test-IM/test-eng.html [hunk 6]
```diff
@@ -216,12 +216,12 @@ <h2 class="wb-inv">About this site</h2>
...
 	</script>
 
 	<!-- Invitation Manager JS Files -->
-	<script src="https://servicecanada.github.io/invitation-manager/src/Overlay.js"></script>
-	<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager.js"></script>
+	<script src="https://ricokola.github.io/invitation-manager/src/Overlay.js"></script>
```
Comment (GormFrank) [inline docs/test-IM/test-eng.html]: Please keep the servicecanada URL, and update the Overlay.js file as needed, and the InvitiationManager.js as needed. We want to track the changes here.

File: docs/test-IM/test-eng.html [hunk 7]
```diff
@@ -216,12 +216,12 @@ <h2 class="wb-inv">About this site</h2>
...
+	<script src="https://ricokola.github.io/invitation-manager/src/InvitationManager-eric.js"></script>
```
Comment (GormFrank) [inline docs/test-IM/test-eng.html]: Idem

File: src/InvitationManager-eric.js [hunk 8]
```diff
@@ -0,0 +1,830 @@
+/*
```
Comment (GormFrank) [inline src/InvitationManager-eric.js]: This file is not needed, please update the InvitationManager.js file as needed instead.

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Comment by Ricokola** [comment]
@GormFrank Propsoed changes have been made and PR has been updated accordingly. Please review and let me konw !

File: docs/test-IM/test-eng.html [hunk 9]
```diff
@@ -1,227 +1,225 @@
...
+		</script>
 
-	<!-- Invitation Manager JS Files -->
-	<script src="https://servicecanada.github.io/invitation-manager/src/Overlay.js"></script>
-	<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager.js"></script>
-	<!-- End Invitation Manager JS Files-->
-</body>
+		<!-- Invitation Manager JS Files -->
+		<script src="https://servicecanada.github.io/invitation-manager/src/Overlay.js"></script>
+		<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager-eric.js"></script>
```
Comment (GormFrank) [inline docs/test-IM/test-eng.html]: This still has `-eric` in the link:
```suggestion
		<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager.js"></script>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
One small remaining change

**Review by GormFrank — APPROVED** [review]

**Pull request merged**